src/js/components/Select/doc.js

```diff
@@ -26,6 +26,23 @@ export const doc = Select => {
     closeOnChange: PropTypes.bool
       .description('Wether to close the drop when a selection is made.')
       .defaultValue(true),
+    clear: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.shape({
+        position: PropTypes.oneOf(['top', 'bottom']).description(
+            `Add a clear value item add the top or at the bottom of the
+            container. By default no unselect option is present.`,
```

Comment:
terminology is confusing when sometimes using `unselect` and `clear` while I think they mean the same.

Reply:
Changing to `clear`